hack/jenkins/release_build_and_upload.sh

```diff
@@ -31,6 +31,11 @@ export DEB_VERSION=${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_BUILD}
 export RPM_VERSION=${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_BUILD}
 export GOPATH=~/go
 
+# Make sure the right golang version is installed based on Makefile
+EXPCECTED_GOLANG_VERSION=$(cat Makefile | grep "GO_VERSION ?=" | awk -F"= " '{print $2}')
```

Comment:
What do you think about `WANT_GO_VERSION`? It seems more Go-like =)

Also, you can skip the cat step... for instance:  `grep '^GO_VERSION' Makefile  | awk '{ print $3 }'`

Reply:
nice trick !